{"uid":"cab72204d4aea6bf","name":"test_landscape_scalable_bundle_db","fullName":"tests.integration.relations.test_db_admin#test_landscape_scalable_bundle_db","historyId":"c1aa2c16b4d501dc360af4888d9bdb43","time":{"start":1755395807276,"stop":1755396810710,"duration":1003434},"description":"Deploy Landscape Scalable Bundle to test the 'db-admin' relation.","descriptionHtml":"<p>Deploy Landscape Scalable Bundle to test the 'db-admin' relation.</p>\n","status":"broken","statusMessage":"TimeoutError: Timed out waiting for model:\n  postgresql/0 [allocating] waiting: agent initialising\n  postgresql/1 [allocating] waiting: agent initialising","statusTrace":"Traceback (most recent call last):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 344, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 246, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 900, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 263, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 178, in pytest_runtest_call\n    item.runtest()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1671, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 157, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/root/spread_project/tests/integration/relations/test_db_admin.py\", line 33, in test_landscape_scalable_bundle_db\n    backend_relation = await deploy_postgres_bundle(\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/tests/integration/helpers/helpers.py\", line 283, in deploy_postgres_bundle\n    await ops_test.model.wait_for_idle(\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 3250, in wait_for_idle\n    raise asyncio.TimeoutError(\nTimeoutError: Timed out waiting for model:\n  postgresql/0 [allocating] waiting: agent initialising\n  postgresql/1 [allocating] waiting: agent initialising","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1755395807130,"stop":1755395807130,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1755395807130,"stop":1755395807275,"duration":145},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"charm","time":{"start":1755395807130,"stop":1755395807130,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1755395807130,"stop":1755395807130,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1755395807275,"stop":1755395807275,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Deploy Landscape Scalable Bundle to test the 'db-admin' relation.","status":"broken","statusMessage":"TimeoutError: Timed out waiting for model:\n  postgresql/0 [allocating] waiting: agent initialising\n  postgresql/1 [allocating] waiting: agent initialising","statusTrace":"Traceback (most recent call last):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 344, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 246, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 900, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 263, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 178, in pytest_runtest_call\n    item.runtest()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1671, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 157, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/root/spread_project/tests/integration/relations/test_db_admin.py\", line 33, in test_landscape_scalable_bundle_db\n    backend_relation = await deploy_postgres_bundle(\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/tests/integration/helpers/helpers.py\", line 283, in deploy_postgres_bundle\n    await ops_test.model.wait_for_idle(\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 3250, in wait_for_idle\n    raise asyncio.TimeoutError(\nTimeoutError: Timed out waiting for model:\n  postgresql/0 [allocating] waiting: agent initialising\n  postgresql/1 [allocating] waiting: agent initialising","steps":[],"attachments":[{"uid":"ed5e14b05f25c1d8","name":"log","source":"ed5e14b05f25c1d8.txt","type":"text/plain","size":22655}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1755396810723,"stop":1755396810723,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1755396810724,"stop":1755396811664,"duration":940},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1755396811665,"stop":1755396811665,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1755396811665,"stop":1755396811665,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1755396811665,"stop":1755396811665,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.relations"},{"name":"suite","value":"test_db_admin"},{"name":"host","value":"pkrvmubgrv54qmi"},{"name":"thread","value":"15019-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.relations.test_db_admin"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"66065e440e49c705","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[]}],"history":{"statistic":{"failed":0,"broken":51,"skipped":0,"passed":299,"unknown":7,"total":357},"items":[{"uid":"3d8ff7217a7e0098","reportUrl":"../2866//#testresult/3d8ff7217a7e0098","status":"passed","time":{"start":1755308181614,"stop":1755309362706,"duration":1181092}},{"uid":"1a321e0842be58ee","reportUrl":"../2865//#testresult/1a321e0842be58ee","status":"passed","time":{"start":1755221775456,"stop":1755222978677,"duration":1203221}},{"uid":"9005cc893b9a2e6a","reportUrl":"../2864//#testresult/9005cc893b9a2e6a","status":"passed","time":{"start":1755135378408,"stop":1755136817343,"duration":1438935}},{"uid":"271d577e2edef9ba","reportUrl":"../2863//#testresult/271d577e2edef9ba","status":"passed","time":{"start":1755048948423,"stop":1755050081353,"duration":1132930}},{"uid":"54f2aa5cb745596","reportUrl":"../2858//#testresult/54f2aa5cb745596","status":"passed","time":{"start":1754963021461,"stop":1754964188205,"duration":1166744}},{"uid":"bf8b8b43014cd881","reportUrl":"../2857//#testresult/bf8b8b43014cd881","status":"broken","statusDetails":"psycopg2.OperationalError: connection to server at \"10.147.162.154\", port 5432 failed: FATAL:  no pg_hba.conf entry for host \"10.147.162.1\", user \"relation-5\", database \"postgres\", no encryption","time":{"start":1754876228114,"stop":1754877339469,"duration":1111355}},{"uid":"2a032fe3e614a43f","reportUrl":"../2856//#testresult/2a032fe3e614a43f","status":"passed","time":{"start":1754789797102,"stop":1754790962290,"duration":1165188}},{"uid":"df30394bd19252dc","reportUrl":"../2855//#testresult/df30394bd19252dc","status":"passed","time":{"start":1754703410668,"stop":1754704601148,"duration":1190480}},{"uid":"8599c20b0e516a7","reportUrl":"../2854//#testresult/8599c20b0e516a7","status":"broken","statusDetails":"psycopg2.OperationalError: connection to server at \"10.126.135.112\", port 5432 failed: FATAL:  no pg_hba.conf entry for host \"10.126.135.1\", user \"relation-5\", database \"postgres\", no encryption","time":{"start":1754617429085,"stop":1754618921179,"duration":1492094}},{"uid":"e47c1a228ed060d7","reportUrl":"../2853//#testresult/e47c1a228ed060d7","status":"passed","time":{"start":1754530605664,"stop":1754531787330,"duration":1181666}},{"uid":"b9006ad6a1906445","reportUrl":"../2852//#testresult/b9006ad6a1906445","status":"passed","time":{"start":1754444418296,"stop":1754446319652,"duration":1901356}},{"uid":"a94a8134917043e9","reportUrl":"../2851//#testresult/a94a8134917043e9","status":"broken","statusDetails":"psycopg2.OperationalError: connection to server at \"10.123.27.76\", port 5432 failed: FATAL:  no pg_hba.conf entry for host \"10.123.27.1\", user \"relation-5\", database \"postgres\", no encryption","time":{"start":1754357876238,"stop":1754359001342,"duration":1125104}},{"uid":"ba68d0f95a554760","reportUrl":"../2850//#testresult/ba68d0f95a554760","status":"passed","time":{"start":1754272110041,"stop":1754273371120,"duration":1261079}},{"uid":"a385ca602c326413","reportUrl":"../2849//#testresult/a385ca602c326413","status":"passed","time":{"start":1754185428211,"stop":1754186584691,"duration":1156480}},{"uid":"791bf9fb39b8f65","reportUrl":"../2848//#testresult/791bf9fb39b8f65","status":"passed","time":{"start":1754098585484,"stop":1754099737412,"duration":1151928}},{"uid":"440f8cd16271fea7","reportUrl":"../2840//#testresult/440f8cd16271fea7","status":"passed","time":{"start":1754012369262,"stop":1754013623197,"duration":1253935}},{"uid":"7454f6c97f0028d5","reportUrl":"../2757//#testresult/7454f6c97f0028d5","status":"passed","time":{"start":1750297525659,"stop":1750298704090,"duration":1178431}},{"uid":"ce7fb3ac1c881b15","reportUrl":"../2753//#testresult/ce7fb3ac1c881b15","status":"passed","time":{"start":1750211107350,"stop":1750212282649,"duration":1175299}},{"uid":"18999cba05f1d268","reportUrl":"../2751//#testresult/18999cba05f1d268","status":"passed","time":{"start":1750124805686,"stop":1750125965725,"duration":1160039}},{"uid":"8c69997bbd755b1b","reportUrl":"../2747//#testresult/8c69997bbd755b1b","status":"passed","time":{"start":1750038263328,"stop":1750039447266,"duration":1183938}}]},"tags":["asyncio"]},"source":"cab72204d4aea6bf.json","parameterValues":[]}